### DISCUSSION TITLE
Как запустить в режиме GUI / TTY

### DISCUSSION THREAD

**Discussion by shmuz** [elfmz/far2l#1187] [Q&A] [answered]
У меня запуск far2l без параметров аналогичен запуску с ключом --notty, и визуально выглядит не как GUI (хотя я не уверен, что это не эмуляция текстового режима в GUI). Если же указать ключ --tty, то вообще не запускается.
Как мне запустить и сравнить оба режима?

**Comment by shmuz** [comment]
Да, похоже у меня режим GUI, ибо при создании папок, например, на китайском, все иероглифы отображаются.
Остаётся вопрос, почему не запускается с --tty.

**Comment by elfmz** [comment, marked as answer]
А в чем запускаете (ось, терминал?). Попробуйте --tty --nodetect

**Comment by shmuz** [comment]
Linux Mint 20 (64 bit) через VirtualBox.
--tty --nodetect ведёт себя также, как просто --tty, а именно:
при двойном клике на иконке Фара выделение иконки пропадает, но окно Фара не появляется.

Вообще режим GUI меня вполне устраивает, но любопытно было бы сравнить с tty.

**Comment by shmuz** [comment]
Всё, получилось. Как вы намекнули, надо было запускать из терминала.
Я новичок в линуксах.
И спасибо за far2l, до него мне было в линуксе совсем плохо.

**Reply by unxed** [reply]
Я, кажется, понял, в чём проблемка могла быть у вас. Когда вы просто пытается запустить из иксов консольную прогу, система не знает, в какой именно терминалке её запустить. Поэтому не `far2l --tty`, а `gnome-terminal -- far2l --tty`, например.

Про то, как печально было без far2l — подписываюсь под каждым словом.

**Comment by shmuz** [comment]
Спасибо, попробую обязательно.
